source/common/rom_db.c

```diff
@@ -53,10 +53,10 @@ ROM_INFO rom_db[] =
     { "Virtual League Baseball (U)",     0x2E20B6E7, "Bad (2)" },
     { "Virtual League Baseball (U)",     0xCE830401, "Bad (3)" },
     { "Virtual Pro Yakyuu '95 (J)",      0x9BA8BB5E, "Good"    },
-    { "Waterworld (U)",                  0x82A95E51, "Good" },
+    { "Waterworld (U)",                  0x82A95E51, "Good"    },
     { "Waterworld (U)",                  0x742298D1, "Bad (1)" },
 
-    { '\0', 0, '\0' }
+    {{ '\0', 0, '\0' }}
```

Comment:
For some reason GCC doesn't like this struct to be bare. I think it's a version of this bug https://gcc.gnu.org/bugzilla/show_bug.cgi?id=53119